File: Dot examples/ASP Book Examples/Book Examples/Chapter18/CS/Standard/App_Code/MovieCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Data.Linq;

public partial class MovieCategory
{
    public static IEnumerable<MovieCategory> Select()
    {
        MyDatabaseDataContext db = new MyDatabaseDataContext();
        return db.MovieCategories;
    }



}

File: Dot examples/ASP Book Examples/Book Examples/Chapter18/VB/Standard/App_Code/MovieCategory.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Data.Linq

Partial Public Class MovieCategory

    Public Shared Function [Select]() As IEnumerable(Of MovieCategory)
        Dim db As New MyDatabaseDataContext()
        Return db.MovieCategories
    End Function



End Class
